packages/connector/src/consts/general.ts

```diff
@@ -1 +1,2 @@
 export const CORE_POLLING_INTERVAL = 5_000;
+export const METAMASK_SNAPS_MAJOR_VERSION = 11;
```

Comment:
Checking the major version is not enough. Let's use something like this to compare with the required version:
https://www.npmjs.com/package/compare-versions